upi/openstack/inventory.yaml

```diff
@@ -32,3 +28,16 @@ all:
       # Number of provisioned Compute nodes.
       # 3 is the minimum number for a fully-functional cluster.
       os_compute_nodes_number: 3
+
+      # If set to 'true', the 'bootstrap' playbook will create a floating IP
+      # attached to the bootstrap node, to enable the Installer to collect a
+      # 'must-gather' log bundle, useful for debugging a failed installation.
+      os_bootstrap_fip: true
+
+      # OpenShift API floating IP address. Will be attached if this value is
+      # non-empty
+      os_api_fip: '203.0.113.23'
```

Comment:
Do you think it is better to leave this as "", or to have a fake FIP in there? Does it get overwritten anyway and not matter?

Reply:
I find it useful that it matches the [documentation]. Is it confusing? I am worried about defaulting to FIPless, but you may have a better idea

[documentation]: https://github.com/openshift/installer/blob/master/docs/user/openstack/install_upi.md#api-and-ingress-floating-ip-addresses

Reply:
I think matching the docs makes sense